### DISCUSSION TITLE
Runtime sizes

### DISCUSSION THREAD

**Discussion by jhgorse** [WasmEdge/WasmEdge#1561] [General]
I was looking at the latest release binaries
https://github.com/WasmEdge/WasmEdge/releases/tag/0.10.0
and noticed that Darwin and Android are a couple of Mb, while Windows and Linux are around 30 Mb.

Why the difference? Guessing here that Darwin and Android might have some native resources for js execution.

Cheers,
Joe

**Comment by hydai** [comment]
The difference is LLVM dependency.

On the Android platform, we do not provide an ahead-of-time compilation mode. So there is no need to link LLVM.
On the Darwin platform, we link LLVM dynamically. So the size is small than other platforms.

**Reply by juntao** [reply]
I would like to add that you can choose to build a small WasmEdge binary for Linux by dynamically linking llvm in your build settings. 

Our default binary build statically links llvm.
